packages/components/tests/dummy/app/templates/components/link/inline.hbs

```diff
@@ -77,7 +77,10 @@
         <li>secondary</li>
       </ol>
     </dd>
-    <dt>"yield"</dt>
+    <dt>
```

Comment:
[nitpick] this technically speaking is not required. it's better if it's provided, of course, but is not `required` per se